lib/Marlin/Marlin/src/feature/prusa/crash_recovery.cpp

```diff
@@ -217,6 +217,7 @@ void Crash_s::update_machine() {
     if(!m_axis_is_homing[0])
     {
         if (enabled) {
+            tmc_enable_stallguard(stepperX);
```

Comment:
Stallguard might trigger falsely at this point, as it is enabled earlier than sensitivity is set, so original sensitivity (homing sensitivity) might be higher than intended for crash detection.
I think there is no possible place to put this function call in this if{} block. E.g. if you put it right after `stepperX.stall_sensitivity(crash_s.sensitivity.x);` it would be OK from the point of sensitivity, but it might trigger falsely because it might set longer `stall_max_period` (TCOOLTHRS) so it might be active even stall_max_period for crash detection prevent its activity at current feedrate.